src/vm/exstatecommon.h

```diff
@@ -51,10 +51,10 @@ class DebuggerExState
         m_pDebuggerContext = NULL;
         m_pDebuggerInterceptNativeOffset = 0;
 
-        // x86-specific fields
-  #if defined(_TARGET_X86_)      
+  #ifndef WIN64EXCEPTIONS
+        // WIN32EXCEPTIONS-specific fields
```

Comment:
There is actually nothing like WIN32EXCEPTIONS. The comment should really stay as it was.